### PR TITLE
Update UG to specify that only 2 commands can be used when viewing archived list

### LINKED ISSUE #228: [PE-D][Tester E] Cannot delete archived user
Deleting an archived user is prohibited by the application. However, considering possible use case as a user, deleting user from archived-list, I believe, seems like a valid use case that should be supported. Although editing might be unnecessary for archived list, deleting from the list could be feature that should be allowed to the user.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: zzawook/ped#8

### PR DESCRIPTION
**Pull request by Dinoman44**
Fixes #228

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ Format: `archive-list`
 
 <div markdown="span" class="alert alert-primary">:information_source: **Note:**
 <br>
-The archive list is a purely read-only list. You can only manipulate it using the command `unarchive`.
+The archive list is a purely read-only list. You can only use two commands while viewing it: `unarchive` (to unarchive a client) and `list` (to navigate back to the main client list).
```
Comment (JunLongling) [inline docs/UserGuide.md:344]: we are able to use exit and help command also

Comment (rahula1008) [reply docs/UserGuide.md:344]: By any chance would it be worth stating that to delete or edit these clients you have to move them back to the main list?

Comment (Dinoman44) [reply docs/UserGuide.md:344]: We can technically also use `archive-list` I just realised; should I add that as well?

Also we can put that in future enhancements that you need to move them back to the main list before editing/deleting, and that we plan to allow it soon.

**Review by Dinoman44 — COMMENTED** [review]
Replied

**Review by rahula1008 — APPROVED** [review]
Approved.

**Pull request merged**